client/components/GuestInviteDrawer/GuestInviteDrawer.js

```diff
@@ -305,21 +317,22 @@ class GuestInviteDrawer extends Component {
           fullWidth
         />
         <div styleName="Row">
-          <FlatButton
+          <RaisedButton
+            styleName="primaryButton"
```

Comment:
The material-UI theme has a primary button style definition its better to call this style with a different name